src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/collector/collector-configuration-examples/opentelemetry-collector-kafka-confluentcloud.mdx

```diff
@@ -66,82 +59,12 @@ Replace the following keys in the file with your own values:
   * bootstrap server provided by confluent for the cluster
   * example: xxx-xxxx.us-east-2.aws.confluent.cloud:9092
 
-```yaml
-receivers:
-  kafkametrics:
-    brokers:
-      - CLUSTER_BOOTSTRAP_SERVER
-    protocol_version: 2.0.0
-    scrapers:
-      - brokers
-      - topics
-      - consumers
-    auth:
-      sasl:
-        username: CLUSTER_API_KEY
-        password: CLUSTER_API_SECRET
-        mechanism: PLAIN
-      tls:
-        insecure_skip_verify: false
-    collection_interval: 30s
-
-
-  prometheus:
-    config:
-      scrape_configs:
-        - job_name: "confluent"
-          scrape_interval: 60s # Do not go any lower than this or you'll hit rate limits
-          static_configs:
-            - targets: ["api.telemetry.confluent.cloud"]
-          scheme: https
-          basic_auth:
-            username: CONFLUENT_API_ID
-            password: CONFLUENT_API_SECRET
-          metrics_path: /v2/metrics/cloud/export
-          params:
-            "resource.kafka.id":
-              - CLUSTER_ID
-exporters:
-  otlp:
-    endpoint: https://otlp.nr-data.net:4317
-    headers:
-      api-key: NEW_RELIC_LICENSE_KEY
-processors:
-  batch:
-  memory_limiter:
-    limit_mib: 400
-    spike_limit_mib: 100
-    check_interval: 5s
-service:
-  telemetry:
-    logs:
-  pipelines:
-    metrics:
-      receivers: [prometheus]
-      processors: [batch]
-      exporters: [otlp]
-    metrics/kafka:
-      receivers: [kafkametrics]
-      processors: [batch]
-      exporters: [otlp]
-
-
-```
-
-## Step 5: Run the collector [#run-collector]
-
-Execute the following, making sure to insert the operating system (for example, `darwin` or `linux`):
-
-```
-./bin/otelcontribcol_INSERT_THE_OPERATING_SYSTEM_amd64 --config config.yaml
-```
-
 
-## Step 6: Set up dashboards in New Relic
+## Step 5: Set up dashboards in New Relic
 
 Check out this New Relic [example dashboard](https://github.com/newrelic/newrelic-quickstarts/blob/main/dashboards/confluent-cloud/confluent-cloud.json) that uses these metrics:
```

Comment:
We'll sometimes give instructions of how users can find this data in our metric explorer. I'm familiar with these steps for infra integrations. How would users filter / query for these metrics from our UI? Is it still from the Metrics explorer? A path and some instructions for them to check that their metrics are reporting could be good